meilisearch/models/task.py

```diff
@@ -19,7 +21,7 @@ class Task(CamelBase):
 
 class TaskInfo(CamelBase):
     task_uid: Optional[str]
-    index_uid: str
+    index_uid: Optional[str]
     status: str
     type: str
     details: Optional[Dict[str, Any]]
```

Comment:
These are the fields present in [`TaskInfo`](https://github.com/meilisearch/specifications/blob/main/text/0060-tasks-api.md#summarized-task-object-for-202-accepted).

I don't think they should be changed in this PR